### ISSUE TITLE
Test: next edit suggestions with a screen reader

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#247331]
Refs #246922, #246921, #230322

- [x] Windows @lszomoru 
- [x] macOS @osortega 
- [x] anyOS @justschen 

Complexity: 3

Authors: @meganrogge, @hediet 

[Create Issue](https://github.com/microsoft/vscode/issues/new?template=blank&body=Testing+%23247331%0A%0A&assignees=meganrogge,hediet)

---

1. **Enable a screen reader** and navigate through the NES experience.
2. Confirm that when a *Next Edit Suggestion* is available, it is announced with both an **sound** and an aria alert **announcement** labeled `"Next Edit Suggestion"`.
3. Verify that those signals can be **configured** using the setting `accessibility.signals.nextEditSuggestion`.
4. Open the **Accessible View** using `Alt+F2`:
   - Confirm that accessibility signals are played for **added (`+`) and deleted (`-`) lines**.
5. **Apply the suggestion** and confirm that it occurred.

**Comment by justschen** [comment]
worked nicely! wondering if there should be an announcment when the suggestion is applied? after hitting tab, the screen reader just started reading random things (ie, i had my tasks tab opened and it started reading the watch tasks and i was confused)

**Comment by meganrogge** [comment]
Good question, thanks for asking @justschen. Given that the user has just accepted the code it should be inserted and the editor should be focused. If that's not the case, that's a bug.
